BraveWallet/Crypto/Transactions/TransactionDetailsView.swift

```diff
@@ -0,0 +1,269 @@
+
+  private var header: some View {
+    TransactionHeader(
+      fromAccountAddress: info.fromAddress,
+      fromAccountName: namedAddress(for: info.fromAddress),
+      toAccountAddress: info.ethTxToAddress,
+      toAccountName: namedAddress(for: info.ethTxToAddress),
+      transactionType: title,
+      value: value,
+      fiat: fiat
+    )
+      .frame(maxWidth: .infinity)
+      .padding(.vertical, 30)
+  }
+  
+  var body: some View {
+    NavigationView {
+      List {
+        Section(
+          header: header
+            .resetListHeaderStyle()
+            .osAvailabilityModifiers { content in
+              if #available(iOS 15.0, *) {
+                content // Padding already applied
+              } else {
+                content.padding(.top)
+              }
+            }
+        ) {
+          if let transactionFee = transactionFee {
+            detailRow(title: Strings.Wallet.transactionDetailsTxFeeTitle, value: transactionFee)
+          }
+          detailRow(title: Strings.Wallet.transactionDetailsMarketPriceTitle, value: marketPrice)
+          detailRow(title: Strings.Wallet.transactionDetailsDateTitle, value: dateFormatter.string(from: info.createdTime))
+          detailRow(title: Strings.Wallet.transactionDetailsTxHashTitle, value: !info.txHash.isEmpty ? info.txHash.truncatedHash : "***")
+          detailRow(title: Strings.Wallet.transactionDetailsNetworkTitle, value: networkStore.selectedChain.chainName)
+          detailRow(title: Strings.Wallet.transactionDetailsStatusTitle) {
+            HStack(spacing: 4) {
+              Image(systemName: "circle.fill")
+                .foregroundColor(info.txStatus.color)
+                .imageScale(.small)
+                .accessibilityHidden(true)
+              Text(info.txStatus.localizedDescription)
+                .foregroundColor(Color(.braveLabel))
+                .multilineTextAlignment(.trailing)
+            }
+            .accessibilityElement(children: .combine)
+            .font(.caption.weight(.semibold))
+          }
+        }
+        .listRowInsets(.zero)
+        if !info.txHash.isEmpty {
+          Section {
+            Button(action: {
+              if let baseURL = self.networkStore.selectedChain.blockExplorerUrls.first.map(URL.init(string:)),
+                 let url = baseURL?.appendingPathComponent("tx/\(info.txHash)") {
+                openWalletURL?(url)
+              }
+            }) {
+              Text(Strings.Wallet.transactionDetailsViewOnEtherscanTitle)
+            }
+            .buttonStyle(BraveFilledButtonStyle(size: .large))
+            .frame(maxWidth: .infinity)
+            .listRowInsets(.zero)
+            .listRowBackground(Color(.braveGroupedBackground))
+          }
+        }
+      }
+      .listStyle(.insetGrouped)
+      .background(Color(.braveGroupedBackground).edgesIgnoringSafeArea(.all))
+      .navigationTitle(Strings.Wallet.transactionDetailsTitle)
+      .navigationBarTitleDisplayMode(.inline)
```

Comment:
I believe `.navigationViewStyle(.stack)` is usually used to prevent a split view on iPad, but from a quick test it seems that the default behaviour of `NavigationView` doesn't behave as a split view when presented modally in a sheet or full screen (when it's not presented as a modal it does default to a split view) 🤔. We use the stack navigation style elsewhere in wallet to prevent against this so I might be missing something; I'm going to add the modifier to be safe and match the other views in wallet.